sfcasts/install.md

```diff
@@ -1,61 +1,41 @@
+
+You probably know the drill: Download the course code from this page and unzip it to find a start directory with the same code that you see here. Check out the `README.md` file for all these setup goodies. I've already done all of these steps except for two. 
+
+Find your terminal and run:
+
+```terminal 
 yarn install
 ```
 
-I actually did run
-that already to save time <affirmative> and then I'll actually compile my assets by
-running
+I ran this already to save time, so I'll actually compile my assets by
+running:
 
-```terminal
+```terminal 
 yarn watch
 ```
 
-You can also run `yarn dev-server`, which is another cool way to do this. Perfect. Then
-I'm gonna open up another tab and say, 
+You can also run `yarn dev-server`, which is another cool way to do this.
+
+Perfect! Then I'm going to open up another tab and say: 
 
-```terminal
+```terminal 
 symfony serve -d
 ```
 
-and that's gonna start a
-local web server using the Symfony binary and it starts at `127.0.0.1:8000`. I'm
-gonna be lazy hold command and click that to say hello to caldron overflow. You've
-been doing our Symfony five series. You're probably familiar with this project, but
-this is not a Symfony 5 project. This is a Symfony 6 project. Ooh, but don't
-worry if you're using Symfony 5, it's not gonna be, it won't be any different.
-
-Now most of the code inside of this project, you don't really need to worry about.
-The most important thing is probably our `src/Entity/` directory. Our doctrine
-entities, our site has questions. Each question has an answer, has a number of
-answers. Each question belongs to a single topic and then we have a user entity. Our
-goal in this tutorial is to create a rich admin section that allows our admin users
-to manage these. So let's get EasyAdmin installed, head terminal and run 
-
-```terminal
+That's going to start a local web server using the Symfony binary and it starts at `127.0.0.1:8000`. I'm gonna be lazy, hold ‘command’ and click that to say “hello” to Cauldron Overflow. If you've been doing our Symfony five series, you're probably familiar with this project, but this is not a Symfony 5 project. This is a Symfony 6 project. But don't worry if you're using Symfony 5. It won't be any different.
+
+Now you don’t really need to worry about most of the code inside this project. The most important thing is probably our `src/Entity/` directory. Our site has questions, and each question has a number of answers. Each question belongs to a single topic and then we have a user entity. Our goal in this tutorial is to create a rich admin section that allows our admin users
+to manage these. So let's get EasyAdmin installed! Find your terminal and run 
```

Comment:
`Now you don’t` -> `You don't` - for 2 small reasons:

A) Same thing as before - avoiding the fancy quote characters and apostrophes
B) Once I read this, the "Now" sounds extra - but subjective and minor

Reply:
This is tricky, but I would do:

> Our site has questions, and each `Question` has a number of answers. Each `Question` belongs
> to a single `Topic`... and then we have a `User` entity.

In this, I'm referring to the `Question` class, hence the upper case and surrounding it by ticks. One trick is that if a file is called `Question.php`, then it ALWAYS holds a class called `Question` inside. So in this case, you don't see the `Question` class (I never open that file), but I'm talking about the class - so upper case + ticks. But, when we says "questions", even though I'm referring to the `Question` class again... it's plural, so there's no proper way to upper case this - saying `Questions` would be wrong, since the class is called `Question`. So, I leave these lower-case.

Reply:
Add a new paragraph between "to manage these." and "So let's get EasyAdmin..."

Reply:
Before a "terminal" block (except in some edge cases where it's totally unnatural due to the way I'm saying the command), we usually put a colon. So:

> Find your terminal and run:

Reply:
> This is tricky, but I would do:
> 
> > Our site has questions, and each `Question` has a number of answers. Each `Question` belongs
> > to a single `Topic`... and then we have a `User` entity.
> 
> In this, I'm referring to the `Question` class, hence the upper case and surrounding it by ticks. One trick is that if a file is called `Question.php`, then it ALWAYS holds a class called `Question` inside. So in this case, you don't see the `Question` class (I never open that file), but I'm talking about the class - so upper case + ticks. But, when we says "questions", even though I'm referring to the `Question` class again... it's plural, so there's no proper way to upper case this - saying `Questions` would be wrong, since the class is called `Question`. So, I leave these lower-case.

I recognized these as classes and thought they might need to be formatted, but I wasn't sure how. Thanks for clearing that up. :)

Reply:
> Before a "terminal" block (except in some edge cases where it's totally unnatural due to the way I'm saying the command), we usually put a colon. So:
> 
> > Find your terminal and run:

I left the ":" off right there intentionally because I had the terminal command as part of that continuing sentence in that edit and breaking it there felt weird. In the updated version I'm working on, I broke it there and started a new sentence after the terminal command, phrased slightly different for optimal flow.